server/src/migrations/1741027685381-ResetMemories.ts

```diff
@@ -0,0 +1,14 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class ResetMemories1741027685381 implements MigrationInterface {
+  name = 'ResetMemories1741027685381';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`DELETE FROM "memories"`);
```

Comment:
Do we need cascade option here? When I run the truncate commands, it needs that

Reply:
Hmm. I tested this and didn't need it so I don't think so

Reply:
Delete should automatically cascade where as truncate won't without explicitly saying `cascade`